src/hidimstat/gaussian_knockoff.py

```diff
@@ -181,7 +181,7 @@ def _s_equi(sigma, tol=1e-14):
         )
         warnings.warn(
             "The equi-correlated matrix for knockoffs is not positive "
-            "definite. Reduce the value of distance.",
+            f"definite. Reduce the value of distance by {s_eps}.",
```

Comment:
I don't understand: what is the user supposed to do when he/she gets this warning ?

Reply:
They cannot do something except use other methods.
The main cause is that we don't provide another way of estimating the diagonal matrix of correlation between real and knockoff variables.